### PR TITLE
Add the OpenTelemetry Collector based on OpenLLMetry to Registry

### PR DESCRIPTION
**Pull request by jinsongo**

### CODE REVIEW

**Comment by jinsongo** [comment]
cc @gyliu513

**Review by theletterf — APPROVED** [review]

**Comment by jinsongo** [comment]
@theletterf @gyliu513 Could you please merge the PR, I am not authorized. Thanks.

**Comment by theletterf** [comment]
We need more approvals before going forward.

File: data/registry/collector-receiver-openllmetry.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# cSpell:ignore openllmetry Instana
+title: IBM OpenTelemetry Collector for LLM
```
Comment (svrnm) [inline data/registry/collector-receiver-openllmetry.yml:2]: The repository title is "OTel based Data Collector for LLM", what is the difference here?

Also "IBM OpenTelemetry Collector for LLM" goes against the trademark requirements for OpenTelemetry, see https://www.linuxfoundation.org/legal/trademark-usage

File: data/registry/collector-receiver-openllmetry.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+registryType: receiver
+language: java
+tags:
+  - java
+  - receiver
+  - collector
+license: Apache 2.0
```
Comment (svrnm) [inline data/registry/collector-receiver-openllmetry.yml:9]: The repository is MIT licensed

File: data/registry/collector-receiver-openllmetry.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+description: IBM OpenTelemetry Collector for LLM based on Instrumentations of OpenLLMetry.
+authors:
+  - name: IBM Instana
+urls:
+  repo: https://github.com/instana/otel-dc/tree/main/llm
+createdAt: 2024-05-07
+package:
+  registry: packagist
```
Comment (svrnm) [inline data/registry/collector-receiver-openllmetry.yml:17]: this is for PHP

File: data/registry/collector-receiver-openllmetry.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# cSpell:ignore openllmetry Instana
+title: IBM OpenTelemetry Collector for LLM
+registryType: receiver
+language: java
+tags:
+  - java
+  - receiver
+  - collector
+license: Apache 2.0
+description: IBM OpenTelemetry Collector for LLM based on Instrumentations of OpenLLMetry.
```
Comment (svrnm) [inline data/registry/collector-receiver-openllmetry.yml:10]: I like the description in the repository more, this is just a long version of the title

File: data/registry/collector-receiver-openllmetry.yml (hunk 1 above)
Comment (jinsongo) [reply data/registry/collector-receiver-openllmetry.yml]: The title has been changed to be same as in repository.

File: data/registry/collector-receiver-openllmetry.yml (hunk 2 above)
Comment (jinsongo) [reply data/registry/collector-receiver-openllmetry.yml]: Yes, it's MIT. Thanks.

File: data/registry/collector-receiver-openllmetry.yml (hunk 3 above)
Comment (jinsongo) [reply data/registry/collector-receiver-openllmetry.yml]: Removed. Thanks.

File: data/registry/collector-receiver-openllmetry.yml (hunk 4 above)
Comment (jinsongo) [reply data/registry/collector-receiver-openllmetry.yml]: The description has been changed to be same as in the repository. Thanks.

**Comment by jinsongo** [comment]
@svrnm Thank you for your detailed advise! Could you please review again.

**Review by svrnm — APPROVED** [review]

**Pull request merged**